src/main/java/com/incognia/feedback/PostFeedbackRequestBody.java

```diff
@@ -3,11 +3,13 @@
 import lombok.Builder;
 import lombok.Value;
 
+import java.time.Instant;
+
 @Value
 @Builder
 public class PostFeedbackRequestBody {
   FeedbackEvent event;
-  Long timestamp;
+  Instant occurredAt;
```

Comment:
This is a breaking change once `PostFeedbackRequestBody` is `public`, but I'm considering it as not because the user isn't supposed to use this class directly. Should change the visiblity of these internal classes to avoid issues?

Reply:
yep

Reply:
I'll change it in another PR to be released together with the breaking changes.